### PR TITLE
Allow to change the number of tasks by environment propriety.

### PR DESCRIPTION
**Pull request by bastiao**
You can change the number of parallel tasks by running:

```
java -Ddicoogle.taskManager.nThreads=8 -jar dicoogle.jar
```

### CODE REVIEW

File: dicoogle/src/main/java/pt/ua/dicoogle/server/ControlServices.java [hunk 1]
```diff
@@ -50,7 +50,7 @@
     
     private ControlServices()
     {
-        TaskManager taskManager = new TaskManager(4);
+        TaskManager taskManager = new TaskManager(Integer.parseInt(System.getProperty("dicoogle.taskManager.nThreads", "4")));
```
Comment (Enet4) [inline dicoogle/src/main/java/pt/ua/dicoogle/server/ControlServices.java]: This looks superfluous. Don't we want a single task manager? This one is just dropped out of scope when leaving the constructor.

Comment (bastiao) [inline dicoogle/src/main/java/pt/ua/dicoogle/server/ControlServices.java]: True. I just did not want to do major refactor at this time.

Comment (Enet4) [inline dicoogle/src/main/java/pt/ua/dicoogle/server/ControlServices.java]: Removing this line should be nothing more than an improvement, not a refactor. :)
If it breaks something, for some reason, we can keep it and address the problem separately.

Comment (bastiao) [inline dicoogle/src/main/java/pt/ua/dicoogle/server/ControlServices.java]: Yes, that's a good idea to address the subject separately 👍

**Pull request merged**